server/data/licenceTypes.ts

```diff
@@ -265,7 +267,7 @@ export interface Risk {
   riskManagement?: RiskManagement
 }
 
-export type RiskVersion = 1 | 2
+export type RiskVersion = '' | '1' | '2'
```

Comment:
maybe undefined would be better than empty string?

Reply:
Sorted in recent commit - agreed that version would be either '1' or '2' and adjusted getRiskVersion method to check for in progress v1 risk management where risk management object is present in licence but version not yet assigned.